src/App.js

```diff
@@ -1,25 +1,26 @@
 import './App.scss';
 import { Route, Routes } from 'react-router-dom';
-import  Navigation  from './components/Navigation/Navigation';
+import Navigation from './components/Navigation/Navigation';
 import Forms from './components/Forms/Forms';
 import ListRestorans from './components/ListRestorans/ListRestorans';
 import Footer from './components/Footer/Footer';
 import RestoranTopFood from './components/Restorans/RestoranTopFood';
-
+import Meni from './components/Restorans/Meni/Meni';
+import Impressions from './components/Restorans/Impressions/Impressions';
+import Galery from './components/Restorans/Galery/Galery';
 const App = () => {
   return (
     <div className='App'>
-       <Navigation /> 
-      <Forms />
-      <ListRestorans />
-      <Footer /> 
-      <RestoranTopFood/>
-      {/* <Routes>
-        <Route path='/navigation' element={<Navigation/>}></Route>
-        <Route path='/forms' element={<Forms/>} />
-        <Route path='/listrestoran' element={<ListRestorans/>} />
-        <Route path='/footer' element={<Footer/>} />
-      </Routes>  */}
```

Comment:
You should remove comments when pushing code to repo

Reply:
Deleted